web/src/components/footer/subcomponents/browse/browse.js

```diff
@@ -0,0 +1,31 @@
+import { Box, Link, Typography } from "@mui/material";
+import browseStyles from "./browse.styles";
+import FeatureFlags from "infra/feature_flags"
+
+const pages = [
+  { name: "Home", to: "#/" },
```

Comment:
We have NavOptions.js - should we move that to somewhere common and re-use those here instead?

Reply:
My qualm with that is that it's a list, so how do we find them? I'd consider it if nav options also had a map
```
const navOptionsMap = {
home: ...
}

const navOptions = navOptionsMap.toList()

...
import navOptionsMap

pages = [navOptionsMap.home, ...]

```